src/excmds.ts

```diff
@@ -839,6 +839,10 @@ function tabSetActive(id: number) {
 //#content_helper
 let JUMPED: boolean
 
+/** @hidden */
+//#content_helper
+let JUMP_TIMEOUTID: number | undefined
```

Comment:
```suggestion
let JUMP_TIMEOUTID
```

This allows me to build without fiddling with tsconfig or the tests.